### ISSUE TITLE
Make it possible for admins to turn off account suggestions

### ISSUE THREAD

**Issue by pezcurrel** [mastodon/mastodon#24474]
### Pitch

I think it would be good if admins could turn off "account suggestions". 
Currently, turning off "Enable trends" switches off trending posts and hashtags, and since accounts are suggested on a "followers count" basis, i think it would be ok if turning off "Enable trends" turned off "account suggestions" too; but a separate setting could be good too.

### Motivation

The possibility for admins to exclude mechanisms which promote competition from their instance's setup.

**Comment by cgubi** [comment]
This would be a great feature to make optional the "algorithmic features" for admins that doesn't feel the need to suggest accounts based on calculations, but just want to promote exploration of the fediverse biodiversity to find relevant content.

**Comment by kenobit8bit** [comment]
I think this would be a very important feature. For many users that signed in on the instance we manage, the absence of "algorithmic features" is very important.

**Comment by eargasmProphet** [comment]
I think that making such a feature _optional_ is **mandatory** ...sorry for the pun. 
A lot of instances around Mastodon have been born with the idea to form communities w/o the sense of competition and/or social hierarchies generated by a popularity score, therefore it is super important to us to keep this chance available for everyone that just wants to be in a virtual space without exploits of that kind.

**Comment by keyezen** [comment]
I agree with this suggestion

**Comment by baselab** [comment]
Yes please make it happen! A lot of communities appeared with no need of such features!

**Comment by sabrinaweb** [comment]
I agree. One of the first things I learned to love about Mastodon is the relaxed environment, compared to the far more competitive birdsite. Mastodon is like a quiet pub where you can chill and relax, not a race to see who's more popular. And please, allow us to keep it this way. Do not impose us features that promote competition, let our admins opt out at least

**Comment by ottomano** [comment]
Why don't implement it at client level, so the choice is up to each user?

**Comment by 128c** [comment]
I agree. Better avoiding competition, gamification and so on.

**Comment by strk** [comment]
> Why don't implement it at client level, so the choice is up to each user?

I think we *are* talking about client level, or are "account suggestions" affecting API usage ?
